### DISCUSSION TITLE
Mirlo Mondays

### DISCUSSION THREAD

**Discussion by simonv3** [funmusicplace/mirlo#747] [Ideas]
To counter Bandcamp Fridays.

Is there a way for purchases on the platform to have the money go entirely to the musicians on a given day?

**Comment by LouisLouiseKay** [comment]
I think anything that moves the needle towards proposing a healthy alternatives to the burnout inducing aspects of Bandcamp Fridays is a good thing.

**Comment by looowizz** [comment]
I think adding this while mirlo is still in early stages and needs the funds is not the best idea, maybe leaving this for a bit is best? Love the idea tho

**Comment by simonv3** [comment]
Letting musicians set their own take rate kind of nullifies this, and we've also started using Mirlo Mondays as a "here's a song from Mirlo" kind of thing
